components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -802,15 +793,31 @@ class ExperimentsTest {
     fun `loading corrupt JSON`() {
         val experimentSource: KintoExperimentSource = mock()
 
-        val file = File(RuntimeEnvironment.application.filesDir, "corrupt-experiments.json")
+        val file = File(ApplicationProvider.getApplicationContext<Context>().filesDir,
+            "corrupt-experiments.json")
         file.writer().use {
             it.write("""{"experiment":[""")
         }
 
         val experimentStorage = FlatFileExperimentStorage(file)
 
-        resetExperiments(experimentSource, experimentStorage)
+        // Because we interact with experimentsUpdater in resetExperiments in order to setup mocking
+        // of the update process, we need to use a scoped down version of the reset here to pass the
+        // verify zero interactions assertion
```

Comment:
Can we use `verifyNoMoreInteractions()`?

Reply:
I cannot use `verifyNoMoreInteractions` here, this comment was relevant to the other place I had to `resetExperiments` minus the updater mocking and I just didn't update it to reflect the use-case.  I updated the comment to try and explain what I was seeing and why.